lib/model/query/datasets.js

```diff
@@ -391,12 +396,39 @@ const getDiff = (projectId, xmlFormId, forDraft) => ({ all }) => all(sql`
   .then(r => Object.keys(r).map(k => ({ name: k.split(',')[0], isNew: forDraft ? k.split(',')[1] === 'true' : undefined, properties: r[k] })));
 
 const update = (datasets, data) => ({ one }) => one(updater(datasets, data)).then(construct(Dataset));
-update.audit = (datasets, data) => (log) => log('dataset.update', datasets, { data });
+update.audit = (datasets, data, autoConvert) => (log) => log('dataset.update', datasets, { data, autoConvert });
+
+const _unprocessedSubmissions = (datasetId, fields) => sql`
```

Comment:
I think deleted submissions (for forms in the trash) should also be excluded. I don't think something in the trash should cause side effects elsewhere in the project.